contrib/ohi-release-notes/run.sh

```diff
@@ -95,6 +98,11 @@ if ! [ -x $RT_BIN ]; then
     help "rt binary is not executable: \"${RT_BIN}\""
 fi
 
+# fetch default dictionary by default
+if [ -z "$DICTIONARY" ]; then
+    DICTIONARY="${TEMP_BIN}/rt-dictionary.yml"
```

Comment:
It was called `TEMP_BIN` because it was meants to be used only for the bin.

if it is going to become a temporary folder I would rename it to `TEMP_DIR`.
```suggestion
    DICTIONARY="${TEMP_DIR}/rt-dictionary.yml"
```